README.md

```diff
@@ -259,3 +284,7 @@ Recording usage can only be performed against already registered measure and and
 
     // multiple measurements can be performed at once.
     stats.Record(ctx, mi.Is(4), mf.Is(10.5))
+
+
+TODO: explain how to access collected data and view.
```

Comment:
have a CL that fixes this. Have been waiting for @rakyll to approve for a couple of weeks :).
Just added as reviewer also.